### DISCUSSION TITLE
Manual Incident

### DISCUSSION THREAD

**Discussion by alexharrington** [upptime/upptime#642] [Ideas]
It would be great to be able to manually add an incident like you can for Scheduled Maintenance, but that shows up as an incident but doesn't count as downtime (unless the check fails and then a new issue would open as normal).

Where I'd like to use that is where we have an issue we know about with a service, and are managing, so there's no downtime, but I'd like to be able to alert my users that there's a potential for an issue, and keep them updated about progress towards a full fix.

At the moment I use a Scheduled Maintenance post for that, but it's not ideal as it says "Scheduled" at the top.

**Comment by Andre601** [comment]
The action adds two labels: `status` and one named after your endpoint name.
Perhaps opening an issue with just the endpoint label could work? Or alternatively just the Status label?

**Reply by alexharrington** [reply]
If you do that it counts it as downtime in the overall figures for that service unfortunately.

**Reply by Andre601** [reply]
Well... In that case should probs a solution similar to scheduled maintenance (As in using a specific label) be made.

We can only wait and see I guess.

**Reply by alexharrington** [reply]
That's what I'm hoping for :crossed_fingers:

**Reply by Andre601** [reply]
Me too, but sadly is AnandChowdhary extremely busy IRL (As mentioned in #618)... And he's pretty much the only person that can implement these things.

I really hope there will be a bit more transparency in the future regarding planned features and such as I'm sure people have an interest in contributing but simply fear what and where they can contribute.

**Reply by AnandChowdhary** [reply]
:'( @Andre601 is right, I'm doing my best but I know for sure that it's not enough. I propose a call for maintainers on the project README and I'd be happy to do a "knowledge transfer" with whoever is up for it? What do you think?
